mathics/core/definitions.py

```diff
@@ -19,14 +21,14 @@
 full_names_pattern = r'(`?{0}(`{0})*)'.format(base_names_pattern)
 
 
-def get_file_time(file):
+def get_file_time(file) -> float:
     try:
         return os.stat(file).st_mtime
     except OSError:
         return 0
```

Comment:
this is an integer

Reply:
But `st_mtime` is a float:

```
>>> os.stat('.zshrc').st_mtime
1548286918.3710866
```

Reply:
and mypy doesn't complain about returning the wrong number type?

Reply:
Yep. I suppose integer literals can be both `int` and `float`.

Reply:
ok :)